package.json

```diff
@@ -60,7 +60,7 @@
     "query-string": "^7.0.0",
     "request": "~2.88.2",
     "semver": "~7.3.5",
-    "simple-icons": "5.1.0",
+    "simple-icons": "5.2.0",
```

Comment:
Could you reset the changes in `package.json` and `package-lock.json`?

```
git checkout master package.json package-lock.json
```

Reply:
All set in [https://github.com/rthartshorn/shields/pull/1/commits/381a0e32ed3ffadc4228f55e85cc8d0090e449fc].